docs/src/content/docs/it/reference/overrides.md

```diff
@@ -50,6 +50,19 @@ Tag di lingua BCP-47 per le impostazioni internazionali di questa pagina, ad es.
 
 Il percorso di base in cui viene servita una lingua. `undefined` per gli slug della lingua di base.
 
+#### `siteTitle`
+
+**Tipo:** `string`
```

Comment:
```suggestion
**tipo:** `string`
```

All the other instances in this page and also in the configuration references seems to be using "tipo" without capitalizing the first letter.

Note that if you think this should be changed, we should do that in a separate PR modifying all files with this pattern maybe.